app/src/main/java/org/jellyfin/mobile/player/PlaybackMenus.kt

```diff
@@ -20,6 +20,8 @@ class PlaybackMenus(private val activity: PlayerActivity) : PopupMenu.OnDismissL
     private val subtitlesButton: View = activity.findViewById(R.id.subtitles_button)
     private val audioStreamsButton: View = activity.findViewById(R.id.audio_streams_button)
     private val infoButton: View = activity.findViewById(R.id.info_button)
+    private val lockScreenButton: View = activity.findViewById(R.id.lock_screen_button)
+    private val unlockScreenButton: View = activity.findViewById(R.id.unlock_screen_button)
```

Comment:
You can move those above the subtitles button to follow the order of the layout (and while you're at it maybe also swap the audio and subtitles buttons, oops)